### PR TITLE
Add Caddy configuration for blocking directory bot

### PR DESCRIPTION
**Pull request by Crocmagnon**
This is the setup I'm currently using.

### CODE REVIEW

File: templates/about.html.tera [hunk 1]
```diff
@@ -33,6 +33,13 @@ RewriteRule . - [R=403,L]
 if ($http_user_agent ~ PrivateBinDirectoryBot ) {
 	return 403;
 }
+			</pre>
+			<h5>Caddy</h5>
+			<pre>
+@privatebinbot {
+    header User-Agent PrivateBinDirectoryBot
```
Comment (elrido) [inline templates/about.html.tera:40]: Given your findings in #19, does this maybe need to use a regex?
```
     header_regexp User-Agent ^PrivateBinDirectoryBot
```

Comment (Crocmagnon) [reply templates/about.html.tera]: I've updated the PR to use a "fast prefix match" according to caddy docs (https://caddyserver.com/docs/caddyfile/matchers#header)

Comment (elrido) [reply templates/about.html.tera]: And it works, it seems. :+1:

**Comment by elrido** [comment]
Thank you!

**Pull request merged**